### ISSUE TITLE
URLCACHE_EXPIRATION_AGE_KEY not working

### ISSUE THREAD

**Issue by gregkerzhner** [evermeer/EVURLCache#19]
The way that the URLCACHE_EXPIRATION_AGE_KEY works is that it looks at the cached file's last modified date, and if that is too long ago, the cached response is not returned.  However, the code in
```
public override func storeCachedResponse(cachedResponse: NSCachedURLResponse, forRequest request: NSURLRequest) 
```
actually always modifies the cached file's last modified date even if the file has not changed and/or no server request has been made.  In other words, every time the function above is called, if an existing cached request is present in the file system, it is rewritten to be the same value, and its last modified date is changed to the current time.   

This is because calling
```
NSKeyedArchiver.archiveRootObject(cachedResponse, toFile: storagePath)
```
inside 'storeCachedResponse' will always update the file's NSFileModificationDate, even if the contents are exactly the same

So, if for example, you set your URLCACHE_EXPIRATION_AGE_KEY to be 20 seconds, but you make a request every 5 seconds, then every 5 seconds the cached response's NSFileModificationDate will be changed to the current time because the file is rewritten with the same already cached response.   Therefore, the cache will keep returning the same old, expired file.

PR for this problem: https://github.com/evermeer/EVURLCache/pull/18

**Comment by evermeer** [comment]
PR is merged, but I can't create a new cocoapod version because of a problem with ReachabilitySwift
See https://github.com/CocoaPods/CocoaPods/issues/178

**Comment by evermeer** [comment]
Problem is fixed. This has now been released as version 2.6.1
